source/MaterialXCore/Document.cpp

```diff
@@ -153,12 +153,22 @@ void Document::initialize()
     setVersionIntegers(MATERIALX_MAJOR_VERSION, MATERIALX_MINOR_VERSION);
 }
 
-NodeDefPtr Document::addNodeDefFromGraph(const NodeGraphPtr nodeGraph, const string& nodeDefName, const string& node,
-                                         const string& version, bool isDefaultVersion, const string& group, string& newGraphName)
+NodeDefPtr Document::addNodeDefFromGraph(const NodeGraphPtr nodeGraph, const string& nodeDefName, const string& category,
```

Comment:
Are these updates to MaterialXCore coupled with the new JavaScript functionality, or could they perhaps be split into their own smaller changelist?

Reply:
I think these can be pulled out.